packages/fixie/src/sidekick-types.ts

```diff
@@ -81,15 +79,15 @@ export interface BaseMessage extends StateFields {
 
 export interface FunctionCall extends BaseMessage {
   kind: 'functionCall';
-  name?: PropsOfComponent<typeof FunctionCallNode>['name'];
-  args?: PropsOfComponent<typeof FunctionCallNode>['args'];
+  name?: string;
```

Comment:
lol, you don't want to pull in an entire dep just to get a type alias for `string`?